dkpro-jwpl-util/pom.xml

```diff
@@ -47,11 +47,6 @@
     <dependency>
       <groupId>com.googlecode.java-diff-utils</groupId>
```

Comment:
Why do we need this dependency here? Can we avoid it?

Reply:
java-diff-utils? It is used somewhere in the code, otherwise the dependency checking plugin would complain. I don't   know what for though.

Reply:
This is required in one of the modules for history diffs.

Reply:
https://github.com/dkpro/dkpro-jwpl/blob/master/dkpro-jwpl-util/src/main/java/org/dkpro/jwpl/util/templates/TextPair.java (difflib packages)